guvnor-project/guvnor-project-client/src/main/java/org/guvnor/common/services/project/client/security/ProjectTreeProvider.java

```diff
@@ -73,12 +73,22 @@ public void setRootNodePosition(int rootNodePosition) {
     public PermissionNode buildRootNode() {
         PermissionResourceNode rootNode = new PermissionResourceNode(i18n.ProjectResource(), this);
         rootNode.setNodeName(i18n.ProjectsNode());
+        rootNode.setNodeFullName(i18n.ProjectsHelp());
         rootNode.setPositionInTree(rootNodePosition);
-        rootNode.addPermission(newPermission(READ), i18n.ProjectActionRead());
-        rootNode.addPermission(newPermission(UPDATE), i18n.ProjectActionUpdate());
-        rootNode.addPermission(newPermission(DELETE), i18n.ProjectActionDelete());
-        rootNode.addPermission(newPermission(BUILD), i18n.ProjectActionBuild());
-        rootNode.addPermission(newPermission(CREATE), i18n.ProjectActionCreate());
+
+        Permission readPermission = newPermission(READ);
+        Permission updatePermission = newPermission(UPDATE);
+        Permission deletePermission = newPermission(DELETE);
+        Permission buildPermission = newPermission(BUILD);
+        Permission createPermission = newPermission(CREATE);
+
+        rootNode.addPermission(readPermission, i18n.ProjectActionRead());
+        rootNode.addPermission(updatePermission, i18n.ProjectActionUpdate());
+        rootNode.addPermission(deletePermission, i18n.ProjectActionDelete());
+        rootNode.addPermission(buildPermission, i18n.ProjectActionBuild());
+        rootNode.addPermission(createPermission, i18n.ProjectActionCreate());
+
+        rootNode.addDependencies(readPermission, updatePermission, deletePermission, buildPermission);
```

Comment:
@dgutierr Thanks.. read the new JavaDoc in the related UF PR and it all makes sense now.